cola/utils_test.py

```diff
@@ -79,41 +79,49 @@ def __init__(self, *cases, ids=None):
 
     @property
     def excluding(self):
-        self.indexing=False
+        self.indexing = False
         self.indexed_cases = set()
         return self
 
     def __getitem__(self, indexed_cases):
-        if isinstance(indexed_cases,tuple) and len(indexed_cases)>1: # multiple arguments, need to use cross product
-            expanded_indexed_cases = [index(c,t) for t,c in zip(indexed_cases,self.cases)]
+        if isinstance(
```

Comment:
this is not better

Reply:
I guess we should just get in the habit of putting the comments above?

Reply:
Yeah good catch. Let me fix this. It is because of the comment indeed. Put it above